src/main/java/spireMapOverhaul/zones/brokenSpace/relics/BrokenSneckoEye.java

```diff
@@ -0,0 +1,40 @@
+package spireMapOverhaul.zones.brokenSpace.relics;
+
+import basemod.helpers.CardModifierManager;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.relics.SneckoEye;
+import spireMapOverhaul.zones.brokenSpace.cardmods.UnreadableCardMod;
+
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class BrokenSneckoEye extends BrokenRelic {
+    public static final String ID = "BrokenSneckoEye";
+    public static final int AMOUNT = 1;
+
+    public BrokenSneckoEye() {
+        super(ID, RelicTier.SPECIAL, LandingSound.CLINK, SneckoEye.ID);
+
+    }
+
+    @Override
+    public void atTurnStart() {
+        flash();
+        for (int i = 0; i < AMOUNT; i++) {
```

Comment:
It's very very niche, but seeing code like this makes me think of an obvious edge case: when the player's deck has only healing cards.

Let's add the following to cover that (at the start of this function):
```
        if (adp().masterDeck.group.stream().allMatch(c -> c.hasTag(AbstractCard.CardTags.HEALING))) {
            return;
        }
```

There are other ways to handle this, but the code I gave seems like the simplest approach that fits in with the current code structure.